### PR TITLE
Gives blind people a chat message when someone tries to handcuff them

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Gives blind people a chat message when someone tries to cuff them.

## Why It's Good For The Game

Currently, blind people get _zero_ feedback that they're being cuffed other than the noise - which is, somewhat ironically, not very disability friendly, and can be easy to miss even if you can hear fine.

It is not something that you should not be able to realise ICly due to lack of sight, as someone cuffing you would be pretty obvious to just about anyone - so it makes no sense to limit it OOCly.

I suppose it is technically a balance change, but it honestly feels more like an oversight than a feature.

## Changelog
:cl:
fix: Blind people now get a chat message when being cuffed.
/:cl:

### CODE REVIEW

File: code/game/objects/items/handcuffs.dm [hunk 1]
```diff
@@ -60,7 +60,8 @@
 	if(!C.handcuffed)
 		if(C.canBeHandcuffed())
 			C.visible_message(span_danger("[user] is trying to put [src.name] on [C]!"), \
-								span_userdanger("[user] is trying to put [src.name] on you!"))
+								span_userdanger("[user] is trying to put [src.name] on you!"), \
+								span_userdanger("You feel someone grab your wrists, the clink of [src.name] rattling around your ears!"))
```
Comment (Ghommie) [inline code/game/objects/items/handcuffs.dm:64]:
```suggestion
								span_userdanger("[user] is trying to put [name] on you!"), \
								span_userdanger("You feel someone grab your wrists, the clink of [name] rattling around your ears!"))
```
There's no need to specify that atom of the var access operation is `src`, it's already implied.

File: code/game/objects/items/handcuffs.dm [hunk 2]
```diff
@@ -60,7 +60,8 @@
 	if(!C.handcuffed)
 		if(C.canBeHandcuffed())
 			C.visible_message(span_danger("[user] is trying to put [src.name] on [C]!"), \
```
Comment (Ghommie) [inline code/game/objects/items/handcuffs.dm:62]:
```suggestion
			C.visible_message(span_danger("[user] is trying to put [name] on [C]!"), \
```

File: code/game/objects/items/handcuffs.dm [hunk 3]
```diff
@@ -59,8 +59,9 @@
 
 	if(!C.handcuffed)
 		if(C.canBeHandcuffed())
-			C.visible_message(span_danger("[user] is trying to put [src.name] on [C]!"), \
-								span_userdanger("[user] is trying to put [src.name] on you!"))
+			C.visible_message(span_danger("[user] is trying to put [name] on [C]!"), \
+								span_userdanger("[user] is trying to put [name] on you!"), \
+								span_userdanger("You feel someone grab your wrists, the clink of [name] rattling around your ears!"))
```
Comment (Fikou) [inline code/game/objects/items/handcuffs.dm:64]: are you sure the third message isnt for everyone blind nearby?

Comment (Ghommie) [reply code/game/objects/items/handcuffs.dm:64]: Oh, fikou's right. Guess you'll have to do something in this style:
```
if(C.isblind())
	C.show_message(span_userdanger("You feel someone grab your wrists, the clink of [name] rattling around your ears!"), MSG_AUDIBLE)
```

Comment (tf-4) [reply code/game/objects/items/handcuffs.dm:64]: Ah, I'll do that in a moment

**Pull request merged**